modules/agar/src/main/ts/ephox/agar/api/PhantomSkipper.ts

```diff
@@ -0,0 +1,16 @@
+import { before } from '@ephox/bedrock-client';
+
+const detect = (): boolean => navigator.userAgent.indexOf('PhantomJS') > -1;
+
+const bddSetup = (): void => {
+  if (detect()) {
+    before(function () {
```

Comment:
If there's a reason this can't be an arrow function it may be worth a comment?

Reply:
It uses `this` so I think it's pretty clear? This would also be flagged by ESLint if it should be an arrow function.

Reply:
🤦‍♂️